docs/it/università-vs-self-made.md

```diff
@@ -0,0 +1,44 @@
+  Inoltre c’è da menzionare che in alcuni ITS viene offerta la possibilità di prepararsi e sostenere esami per certificazioni esterne quali: [CISCO IT Essentials](https://www.netacad.com/courses/os-it/it-essentials), [ECDL](https://www.aicanet.it/aica/qualita/ecdl-full-standard) e [PET/FIRST/ecc.](https://www.cambridgeenglish.org/exams-and-tests/) per citarne solo alcune.
+
+- **Secondo Anno**: Dopo una breve fase di lezioni dedicate agli ultimi moduli, mirate a esplorare concetti più avanzati, come l'_Architettura del Software_ per esempio, l'attenzione si sposta principalmente sullo **stage lavorativo**, nonché ultima attività prima dell’esame finale (necessario per conseguire un _Diploma di [V livello EQF](https://europa.eu/europass/it/description-eight-eqf-levels)_).
+
+**_“Come posso trovare gli ITS?”_**
+
+Prima di rispondere a questa domanda, è bene precisare che gli ITS in Italia offrono una vasta gamma di corsi dedicati a settori diversi, tra cui tecnologia, industria, e servizi. Per ovvi motivi in questo libro, più precisamente in questo capitolo, si parlerà di quelli legati al settore IT.
+
+Chiusa questa parentesi, il sito dell’**_[INDIRE](https://www.indire.it/progetto/its-istituti-tecnici-superiori/dove-sono-gli-its/)_** (_Istituto Nazionale Documentazione Innovazione Ricerca Educativa_) è sicuramente la migliore risorsa per ottenere una risposta a questa domanda. Ovviamente non l’unica, ma sicuramente tra tutte le pagine risulta essere la più completa, presentando anche a sostegno i dati relativi agli ITS della regione e all’occupazione degli studenti diplomati.
+
+**“_Perché l’ITS potrebbe risultare un’ottima via?”_**
+
+Innanzitutto come già citato è un corso molto breve (2 anni), a numero chiuso (quindi con un test di ammissione) e alla portata di tutti dal punto di vista economico, dal momento che l’iscrizione è gratuita o a costo molto ridotto. Solitamente viene fornito un PC da poter utilizzare durante le lezioni e dedicato completamente alla programmazione, ma soprattutto, ciò che contraddistingue l’ITS (e il motivo per cui viene citato in questo capitolo) è la sua **dualità tra lezioni in aula e stage in azienda** per poter testare in modo pratico “sul campo” i concetti appresi.
+
+È vero, la preparazione concettuale e teorica potrebbe risultare meno solida rispetto a quella universitaria, ma è sicuramente più vantaggiosa in ambito aziendale, essendo maggiormente orientata al mondo del lavoro. Proprio in questo contesto, sfruttando le opportunità lavorative e gli incontri con i professionisti del settore, dobbiamo far emergere la nostra proattività e curiosità per valorizzare al meglio le competenze di base che abbiamo acquisito!
+
+Per esempio, le lezioni di Python che ho seguito in Accademia hanno solleticato la mia curiosità e adesso non vedo l'ora di affinare le mie conoscenze su questo linguaggio. Partendo da questa "infarinatura" posso apprendere più facilmente, anche in maniera autonoma.
```

Comment:
Forse mi sarei tenuta sul generico, senza entrare troppo nel "soggettivo".

```suggestion
Per esempio, è possibile fruire di lezioni di Python...
```

Reply:
Capisco l'osservazione, che ci sta, e motivo il perché ho scelto questa forma. Come per il discorso dei "moduli" che variano da corso a corso, anche in questo caso ho voluto fare un esempio pratico, citando in stile esperienza personale (come se lo stessi raccontando al lettore), un linguaggio che non è detto venga effettivamente trattato. Ovvio che se preferiamo tenere il tutto un po' più "oggettivo" posso tranquillamente rivedere la forma.